playbooks/roles/ocp-config/defaults/main/main.yaml

```diff
@@ -12,3 +12,6 @@ chronyconfig:
    enabled: true
 
 cni_network_provider: OpenshiftSDN
+
+clusterNetwork_CIDR: 10.128.0.0/14
+serviceNetwork: 172.30.0.0/16
```

Comment:
Change var names to `cluster_network_cidr` and `service_network` (at all places)